src/Bindings/ManualBindings_Network.cpp

```diff
@@ -878,11 +878,11 @@ static int tolua_cUDPEndpoint_Send(lua_State * L)
 
 	// Get the data to send:
 	AString Data, RemotePeer;
-	int RemotePort = 0;
+	UInt16 RemotePort = 0;
 	S.GetStackValues(2, Data, RemotePeer, RemotePort);
 
 	// Check the port:
-	if ((RemotePort < 0) || (RemotePort > USHRT_MAX))
+	if ((RemotePort < 0) || (RemotePort > std::numeric_limits<UInt16>::max()))
```

Comment:
Oh yeah!

Then in that case, when are the checks in GetStackValues triggered? Is it something to do with a difference between lua_Number and the template type T? Is lua_Number bigger?